company/okrs/2020_q1.md

```diff
@@ -0,0 +1,69 @@
+# 2020-Q1 OKRs
+
+## CEO
+
+1. **CEO: Grow ARR consistently**
+   1. CEO KR: [$N][N] net new ARR
+   1. CEO KR: Sales leadership owns revenue targets
+   1. CEO KR: Sales, engineering, and product have a successful process for prioritization
+1. **CEO: Aspirational, efficient all-remote team**
+   1. CEO KR: Roles and responsibilities are documented and maintained in handbook
+   1. CEO KR: Handbook-first
+
+## Sales and marketing
+
+1. **Sales (Closing Deals): Achieve [$N][N] net new ARR.**
+   1. Sales KR: Have each AE close 3 Tier 1 deals (>=$100K ARR) and 10 non-Tier 1 deals, 2 of which are from their own outbound outreach efforts.
+   1. Sales KR: Get 70% of companies on a free trial to convert to paid customers.
+1. **Sales (Pipeline Optimization): Scale our sales pipeline to 50+ deals per AE, of which 60% of deals are engaged and actively evaluating Sourcegraph.**
+   1. Sales KR: Round robin new instances across team to collectively achieve a 15%+ response rate from Prospects who set up a new Sourcegraph instance.
+
+1. **VP Business: Build a Customer Success organization that supports and helps us exceed our ARR goals.**
+   1. VP Business (Success) KR: Generate at least [$M][M] in net new ARR from existing accounts.
+   1. VP Business (Success) KR: Hire a Head of Customer Success who can own this function.
+   1. VP Business (Success) KR: Reach monthly NPS of 60 at all Tier 1 customers.
+   
+1. **VP Business: Make Sourcegraph a lean organization that uses budgets to guide decision making.**
+
+1. **VP Marketing: Deliver sufficient MQLs to sales to reach [$N][N] net new ARR goal.**
+1. **VP Marketing: Create new category of developer tool.**
+1. **VP Marketing: Become the dominant provider of source code search.**
+
+## Product
+
+1. **Product: WIP Deliver on product roadmap**
+   1. Product KR: Roadmap communicates rough timeline for upcoming work.
+   1. Product KR: Build out per-team strategy RFCs to buffer roadmap.
+   1. Product KR: Dogfooding each feature release to ensure quality UX.
+1. **Product: WIP Reduce product friction**
+   1. Product KR: Reduce time to "aha" (0 to search).
+   1. Product KR: Reduce time 0 to deploy.
```

Comment:
@christinaforney How are you defining time-to-deploy? Either link to a dashboard (if available yet) or a doc describing how this will be measured.

And how does this differ from the previous point (time-to-aha)? Is one for users and one for site admins?

Reply:
Consolidated to single KR of reducing the time from intent to rollout. @beyang - I essentially grabbed one of the distribution KRs where we had overlap since this is a broader issue to tackle from a product perspective. From what we talked about in the team syncs the distribution team will be taking a piece of this by focusing on improving the deployment/rollout process. Does this line up with your expectation from those conversations?